tools/schematics/dt-component/files/libs/barista-components/__name@dasherize__/barista.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "",
+  "description": "",
+  "postid": "<%= dasherize(name) %>",
+  "identifier": "",
+  "category": "components",
+  "public": false,
```

Comment:
why not public?

Reply:
I thought not public as the default would be a good idea. Fixed it though.